### ISSUE TITLE
Add more sound types, other than Organ and 8-bit

### ISSUE THREAD

**Issue by vinit-agr** [billgathen/pentaphone#3]
I need to look in to the code to understand if adding more sound types is currently modular or hard-coded. Is there any direct way to do this right now. if not, can you please suggest the part of the code i should be looking in to add more sound types.

Thanks

**Comment by billgathen** [comment]
The sound types are wrapped in the tones variable in `src/js/components/note.js`.

I had intended to add other sounds and map them to `o` and `p`, but those two sounds were straightforward because they are pure oscillator types (sine and square). To add a guitar sound or a better organ tone would require adding a more-complicated object that incorporated things like effects, etc. The object would probably be referred to in the `osc` function in that same file.

It's definitely doable but I never got around to investigating it further. The [to-do page](https://github.com/billgathen/pentaphone/blob/master/to-do.md) has a link at the bottom to a page explaining how to get a sound more like Jon Lord's Hammond organ tone in Deep Purple, which was going to be the first thing I tackled.

Let me know what you come up with!
